[1차]-다트-게임.js

```diff
@@ -0,0 +1,19 @@
+function solution(dartResult) {
+    let newResult = dartResult
+        .replace(/\*/g, " *2 ")
+        .replace(/S/g, "**1 ")
+        .replace(/D/g, "**2 ")
+        .replace(/T/g, "**3 ")
+        .replace(/\#/g, "(-1) ")
+        .replace(/ \(/g, "(")
+        .replace(/ {1,}\*/g, "*")
+        .replace(/ {1,}$/g, "");
+    let newArray = newResult.split(" ");
```

Comment:
```suggestion
    const newArray = dartResult
        .replace(/\*/g, " *2 ")
        .replace(/S/g, "**1 ")
        .replace(/D/g, "**2 ")
        .replace(/T/g, "**3 ")
        .replace(/\#/g, "(-1) ")
        .replace(/ \(/g, "(")
        .replace(/ {1,}\*/g, "*")
        .replace(/ {1,}$/g, "")
        .split(" ");
```
어처피 메서드를 중첩해서 쭉 써줬기때문에 split()도 함께 써주는게 더 좋을거 같고 참조타입은 const로 선언해주세요